src/table.js

```diff
@@ -125,9 +126,11 @@ function initialize(
   function appendRow(d, i) {
     const itr = tr.cloneNode(true);
     const input = inputof(itr);
-    input.onclick = reselect;
-    input.checked = selected.has(i);
-    input.value = i;
+    if (selectable) {
```

Comment:
Could we check if input is null here instead? Or should we check selectable before calling inputof?

I’m guessing there are other places we need to check if inputof returns null, such as reinput which is called on resort.

Reply:
yes that's better than adding the empty span: PR updated